src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+:root {
+  --colour: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  align-self: center;
+}
+
+.logo_link {
+  display: flex;
+}
+
+.nav_list {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  cursor: pointer;
+}
+
+.nav_item:not(:first-child) {
+  margin-left: 20px;
+}
```

Comment:
No need to fix it - but a cleaner solution will be 

```suggestion
.nav_item {
  cursor: pointer;
   margin-left: 20px;
}

.nav_item:first-child {
  margin-left: 0;
}
```

Notice - it is a bit simpler as you really don't really need `:not` here